CONTIRBUTING.md

```diff
@@ -0,0 +1,129 @@
+to implement a change, you can ask for help or guidance on solving a problem, or
+just let us know you are working on it.
+
+A maintainer will mark the issue as assigned to you to avoid other people
+unknowingly working on the same thing. If you notice an old assigned issue still
+has no update, please still ask. We can reassign as necessary.
+
+This project sometimes has longer form
+[discussions](https://github.com/allan2/dotenvy/discussions). Feel free to
+comment on existing threads or open your own.
+
+## Pull Requests
+
+Pull Requests are the way concrete changes are made to the code, documentation,
+and dependencies in the dotenvy repository.
+
+Even tiny pull requests (e.g., one character pull request fixing a typo in API
+documentation) are greatly appreciated. Before making a large change, it is
+usually a good idea to first open an issue describing the change to solicit
+feedback and guidance. This will increase the likelihood of the PR getting
+merged.
+
+### Tests
+
+If the change being proposed alters code (as opposed to only documentation for
+example), it is either adding new functionality to a crate or it is fixing
+existing, broken functionality. In both of these cases, the pull request should
+include one or more tests to ensure that the crate does not regress in the
+future.
+
+### Discuss and update
+
+You will probably get feedback or requests for changes to your Pull Request.
+This is a big part of the submission process so don't be discouraged! Some
+contributors may sign off on the Pull Request right away, others may have more
+detailed comments or feedback. This is a necessary part of the process in order
+to evaluate whether the changes are correct and necessary.
+
+Any community member can review a PR and you might get conflicting feedback.
+Keep an eye out for comments from code owners to provide guidance on conflicting
+feedback.
+
+#### Commit Squashing
+
+In most cases, do not squash commits that you add to your Pull Request during
+the review process. When the commits in your Pull Request land, they may be
+squashed into one commit per logical change. Metadata will be added to the
+commit message (including links to the Pull Request, links to relevant issues,
+and the names of the reviewers). The commit history of your Pull Request,
+however, will stay intact on the Pull Request page.
+
+### CI
+
+Our continuous integration workflow checks all pull requests to ensure:
+
+- All tests pass on stable and [MSRV]
+- No [clippy](https://github.com/rust-lang/rust-clippy) errors or warnings
+- [Rustfmt](https://github.com/rust-lang/rustfmt) is adhered to
+- [Rustdoc](https://doc.rust-lang.org/rustdoc/write-documentation/linking-to-items-by-name.html)
+  links work.
+
+Before submitting a PR, it may be worth running these checks yourself.
+
+#### MSRV
+
+Check the [readme][MSRV] for the current
+minimum supported rust version. We are open to updating it.
```

Comment:
```suggestion
#### MSRV

Check the [readme][MSRV] for the current
minimum supported rust version. We are open to updating it.
```

I think the MSRV policy in the README and the badge are visible enough to leave this out.

Reply:
Good point. Someone reading the contribution guide has probably already read the readme.